### PR TITLE
Fixed multiseries prediction interval labels

### LINKED ISSUE #4378: Multiseries prediction interval labels includes target name twice
When generating the prediction interval labels for multiseries pipelines with STL decomposers, a KeyError is generated. This is due to the function trying to find a key that includes the target name twice (e.g. {{target|target|series_a}} instead of {{target|series_a}}).

### PR DESCRIPTION
**Pull request by christopherbunn**
Resolves #4378

### CODE REVIEW

**Comment by machineFL** [comment]
No linked issues found. Please add the corresponding issues in the pull request description. <br/> [Use GitHub automation to close the issue when a PR is merged](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

File: evalml/tests/pipeline_tests/regression_pipeline_tests/test_multiseries_regression_pipeline.py [hunk 1]
```diff
@@ -356,7 +356,7 @@ def test_time_series_pipeline_get_prediction_intervals(
     )
     mock_transform_return_value = (
         mock_X,
-        pd.DataFrame(np.random.rand(13, 5)),
+        mock_y,
```
Comment (jeremyliweishih) [inline evalml/tests/pipeline_tests/regression_pipeline_tests/test_multiseries_regression_pipeline.py:359]: anyway we can add coverage here?

Comment (jeremyliweishih) [reply evalml/tests/pipeline_tests/regression_pipeline_tests/test_multiseries_regression_pipeline.py:359]: maybe [here](https://github.com/alteryx/evalml/blob/main/evalml/tests/component_tests/decomposer_tests/test_stl_decomposer.py#L151)?

**Review by eccabay — APPROVED** [review]

**Review by jeremyliweishih — APPROVED** [review]
LGTM

**Pull request merged**